neutrino.go

```diff
@@ -753,7 +753,7 @@ func NewChainService(cfg Config) (*ChainService, error) {
 	if s.persistToDisk {
 		cfg := &chanutils.BatchWriterConfig[*filterdb.FilterData]{
 			QueueBufferSize:        chanutils.DefaultQueueSize,
-			MaxBatch:               1000,
+			MaxBatch:               10,
```

Comment:
Won't this significantly reduce performance on fast systems? Since we now need to commit 100 times more often to disk, always potentially hitting a file system sync?
If this indeed fixes a problem in a slow environment, that makes me think we should make the value configurable through the `lnd` config. That way mobile users can tune the value down while Neutrino users on powerful machines would leave the original value.

Reply:
In theory yes but we tested on a desktop neutrino node and observed that in practice, batch sizes were often 1.

So 10 gives us some breathing room.

Sync times were not affected.

This only sets the cap anyway, I think the root of the problem is actually how we determine batch size at runtime.